### PR TITLE
test: commit integration tests by georgiyekkert

### PR DESCRIPTION
**Pull request by vam-google**
Merge https://github.com/googleapis/java-compute/pull/290 overwriting history, since the original PR cannot be merged. Tests are by @georgiyekkert

### CODE REVIEW

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/BaseTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Google LLC
```
Comment (chingor13) [inline google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/BaseTest.java:2]:
```suggestion
 * Copyright 2021 Google LLC
```

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITAddressesTest.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.compute.v1.integration;
+
+import static org.junit.Assert.fail;
+
+import com.google.cloud.compute.v1.*;
```
Comment (chingor13) [inline google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITAddressesTest.java:20]: We shouldn't be using wildcard imports.

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2020 Google LLC
```
Comment (chingor13) [inline google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java:2]:
```suggestion
 * Copyright 2021 Google LLC
```

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.compute.v1.integration;
+
+import com.google.cloud.compute.v1.*;
```
Comment (chingor13) [inline google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java:18]: same here on imports

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java [hunk 5]
```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright 2020 Google LLC
```
Comment (chingor13) [inline google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java:2]:
```suggestion
 * Copyright 2021 Google LLC
```

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.compute.v1.integration;
+
+import static junit.framework.TestCase.fail;
+
+import com.google.api.gax.rpc.InvalidArgumentException;
+import com.google.api.gax.rpc.NotFoundException;
+import com.google.cloud.compute.v1.*;
```
Comment (chingor13) [inline google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java:22]: same here on imports

**Review by chingor13 — CHANGES_REQUESTED** [review]

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/BaseTest.java (hunk 1 above)
Comment (vam-google) [reply google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/BaseTest.java:2]: These files were actually created in 2020, that is why I decided to keep 2020. But ok, probably the year of the push must be the year.

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITAddressesTest.java (hunk 2 above)
Comment (vam-google) [reply google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITAddressesTest.java:20]: Oh, yes absolutely. Fixed.

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java (hunk 3 above)
Comment (vam-google) [reply google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java:2]: Fixed

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java (hunk 4 above)
Comment (vam-google) [reply google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java:18]: Fixed

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java (hunk 5 above)
Comment (vam-google) [reply google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java:2]: Fixed

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java (hunk 6 above)
Comment (vam-google) [reply google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java:22]: Fixed

**Comment by vam-google** [comment]
@chingor13 PTAL

**Review by chingor13 — APPROVED** [review]

**Pull request merged**